index.js

```diff
@@ -60,6 +60,30 @@ const wrapWord = (rows, word, columns) => {
 	}
 };
 
+// Trims spaces from a string ignoring invisible sequences
+const stringVisibleTrimSpaces = str => {
+	const words = str.split(' ');
+	let first = 0;
+	let last = words.length;
+	while (first < last) {
+		if (stringWidth(words[first]) > 0) {
+			break;
+		}
+
+		first++;
```

Comment:
The tests never hit this line.  Is it possible for `words.length > 0 && stringWidth(words[0]) === 0`?

Reply:
Yeah, in actual code left side is always trimmed already. Changed to trimRight